tests/test_route_map.rb

```diff
@@ -340,8 +346,9 @@ def test_match_ipv6_multicast
   end
 
   def test_match_metric
-    # bug on fretta
-    skip('platform not supported for this test') if node.product_id[/N9.*-F/]
+    # bug CSCvc82934 on fretta and n9k running dplus
+    skip('platform not supported for this test') if node.product_id[/N9.*-F/] ||
```

Comment:
Would it be more appropriate to use `skip_legacy_defect` instead and list the ddts?